config/config.go

```diff
@@ -140,6 +140,11 @@ func GetDefaultPluginPath() string {
 	return defaultPluginsPath
 }
 
+// returns default plugin version of 1.0.0
+func GetDefaultPluginVersion() string {
+	return "1.0.0"
```

Comment:
as we discussed before, my concern is that if `1.0.0` applies to all external plugins as default version

Reply:
add in doc that we default to 1.0.0 . TODO : update our CRD doc to mention the default value

Reply:
doc PR at https://github.com/deislabs/ratify-web/pull/50

Reply:
added comment for specifying desired version in plugin config